test/unit/core/src/BoostPredefTest.cpp

```diff
@@ -31,9 +31,12 @@ TEST_CASE("printDefines", "[core]")
 #if BOOST_COMP_NVCC
     std::cout << "BOOST_COMP_NVCC:" << BOOST_COMP_NVCC << std::endl;
 #endif
-#if BOOST_COMP_HCC
+#if BOOST_COMP_HCC || BOOST_COMP_HIP
```

Comment:
Is it intended that for HIP now both lines with `BOOST_COMP_HCC:` and `BOOST_COMP_HIP:` will be printed?

Reply:
thx must be removed